Onepunch/color.ini

```diff
@@ -1,7 +1,39 @@
+[wal]
```

Comment:
I'm not sure this theme is needed. It does not add anything by itself. You should add the information in the comments in the README as this would also make it easier to find it

Reply:
was thinking of adding a shell script. but I think you are right. it's useless and can break the theme for most of the color scheme